src/main/java/com/synopsys/integration/alert/web/security/authentication/UserManagementAuthoritiesPopulator.java

```diff
@@ -72,19 +69,10 @@ public UserManagementAuthoritiesPopulator(AuthenticationDescriptorKey Authentica
     }
 
     public Set<String> addAdditionalRoleNames(String userName, Set<String> existingRoles, boolean appendRolePrefix) {
-        // TODO remove in 6.0.0 the createRolesMapping method.
-        Map<String, String> roleMap = createRolesMapping(appendRolePrefix);
         Set<String> rolesFromDB = getRolesFromDatabase(userName, appendRolePrefix);
         Set<String> roles = new LinkedHashSet<>();
         roles.addAll(rolesFromDB);
-        roles.addAll(existingRoles.stream().filter(role -> StringUtils.isNotBlank(role)).collect(Collectors.toSet()));
-        //TODO remove in 6.0.0 with the deprecated method
-        if (!roleMap.isEmpty()) {
-            roles.addAll(existingRoles.stream()
-                             .filter(roleMap::containsKey)
-                             .map(roleMap::get)
-                             .collect(Collectors.toSet()));
-        }
+        roles.addAll(existingRoles.stream().filter(StringUtils::isNotBlank).collect(Collectors.toSet()));
```

Comment:
Should the style be like the following to match our convention with streams()?
`
roles.addAll(existingRoles
                .stream()
                .filter(StringUtils::isNotBlank)
                .collect(Collectors.toSet()));

`

Reply:
You can use this for multi-line code blocks.
 \`\`\`java 
<your_code>
\`\`\`